src/main/java/teammates/ui/webapi/Action.java

```diff
@@ -86,6 +86,10 @@ public void setLogsProcessor(LogsProcessor logsProcessor) {
         this.logsProcessor = logsProcessor;
     }
 
+    public void setLogic(Logic logic) {
```

Comment:
Why is this needed?

Reply:
I need to use `getFeedbackSessionLogs` from LogicExtension (which use `getAllFeedbackSessionLogs`)